src/ui/components/JEXLDebuggerPage.tsx

```diff
@@ -109,15 +163,19 @@ const JEXLDebuggerPage: FC = () => {
                 {["number", "string", "boolean"].includes(typeof value) ? (
```

Comment:
because we overwrite the original context, we lose the original type information. We should keep the two contexts separate.